addon/components/o-s-s/avatar.ts

```diff
@@ -16,6 +16,8 @@ interface OSSAvatarArgs {
   image?: string;
   initials?: string;
   size?: SizeType;
+  loading?: boolean;
+  isPlaceholder?: boolean;
```

Comment:
Is it really a placeholder here? When it is true, you add a class with a background color

Reply:
don't have any good name 😅  ahah